modules/post-installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-configuring-nutanix-failure-domains_{context}"]
+= Adding failure domains to the Infrastructure CRD
+
+You add failure domains to an existing Nutanix cluster by modifying its Infrastructure custom resource definition (CRD) (`infrastructures.config.openshift.io`).
+
+.Procedure
+
+. Edit the Infrastructure CRD by running the following command:
++
+[source,terminal]
+----
+$ oc edit infrastructures.config.openshift.io cluster
+----
+
+. Configure the failure domains.
++
+.Example Infrastructure CRD with Nutanix failure domains
+[source,yaml]
+----
+spec:
+  cloudConfig:
+    key: config
+    name: cloud-provider-config
+#...
+  platformSpec:
+    nutanix:
+      failureDomains:
+      - cluster:
+         type: UUID
+         uuid: <uuid>
+        name: <failure_domain_name>
+        subnets:
+        - type: UUID
+          uuid: <network_uuid>
+      - cluster:
+         type: UUID
+         uuid: <uuid>
+        name: <failure_domain_name>
+        subnets:
+        - type: UUID
+          uuid: <network_uuid>
+# ...
+----
+where:
+
+`<uuid>`:: Specifies the universally unique identifier (UUID) of the Prism Element.
+`<failure_domain_name>`:: Specifies a name for the failure domain.
+`<network_uuid>`:: Specifies the UUID of the Prism Element network that contains the virtual IP addresses and DNS records that you configured. Only one subnet per {product-title} cluster is supported.
```

Comment:
Only one subnet per Failure Domain is supported.

Reply:
Done.